### ISSUE TITLE
[Feature][Future] UI for the app.

### ISSUE THREAD

**Issue by surister** [Pringleman83/SportsBook#61]
Once the project gets bigger it might be interesting to make an UI for it, so it'd go from being a CLI (command line interface) app to a full featured desktop app :P!.

**Comment by Pringleman83** [comment]
Yeah, I was thinking about this. It definitely needs to happen. I have a lot of ideas for the app. I'll get them together and post them when I can.
